collector/docker_proxy.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+#
+# Copyright 2015 The Cluster-Insight Authors. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import datetime
+import flask
+from flask_cors import CORS
+import json
+import logging
+import requests_unixsocket
+import sys
+import types
+
+import constants
+
+app = flask.Flask(__name__)
+logger = logging.getLogger(__name__)
+
+
+# enable cross-origin resource sharing (CORS) HTTP headers on all routes
+cors = CORS(app)
+
+# Start unix socket session
+session = requests_unixsocket.Session()
+
+# Constant pointing to the url for the docker unix socket
+LOCAL_DOCKER_HOST= 'http+unix://%2Fvar%2Frun%2Fdocker.sock'
+
+
+def make_error(error_message):
```

Comment:
This code is duplicated. Please move it to utilities.py so you can call it from both collector.py and from this file.